web.py

```diff
@@ -174,7 +174,7 @@ def GET_v1_healthcheck():
         'last_ping': qmk_redis.get('qmk_api_last_ping'),
         'queue_length': len(rq),
         'queued_job_ids':  rq.job_ids,
-        'queued_jobs': rq.jobs,
+        'queued_jobs': repr(rq.jobs),
```

Comment:
I told him to add that because in some instances it was a bug. I've actually been planning to remove the `queued_jobs` key.

Reply:
Yeah, at least in my case what was being returned was nothing and an exception was being raised. Lol

Reply:
The JSON encoder likely couldn't serialize the object(s) — makes sense. I think a short comment should be added for clarification for now, until the key is removed. Something like `  # Convert to string so it can be serialized.` should do.

Reply:
No comment is needed. It's actually already removed in my local tree, which will bubble out in an upcoming change.